WcaOnRails/db/migrate/20160505231300_create_registration_events.rb

```diff
@@ -0,0 +1,19 @@
+class CreateRegistrationEvents < ActiveRecord::Migration
+  def change
```

Comment:
I've just run this in the production:
```ruby
irb(main):002:0> Registration.all.pluck(:eventIds).map { |e| (e || "").split }.flatten.uniq
   (208.9ms)  SELECT `Preregs`.`eventIds` FROM `Preregs`
=> ["333", "444", "555", "222", "333oh", "333fm", "minx", "pyram", "sq1", "clock", "666", "777", "magic", "333bf", "333mbf", "mmagic", "444bf", "333ft", "555bf", "skewb"]
```
So moving the data should be pretty straightforward =)